Dockerfile

```
@@ -1,24 +1,25 @@
 ARG MINIFORGE_VERSION=22.9.0-2
-
 FROM condaforge/mambaforge:${MINIFORGE_VERSION} AS builder
-
 # Use mamba to install tools and dependencies into /usr/local
 ARG SAMTOOLS_VERSION=1.21
 RUN mamba create -qy -p /usr/local \
-    -c bioconda \
     -c conda-forge \
+    -c bioconda \
```

```suggestion
    -c conda-forge \
    -c bioconda \
    --strict-channel-priority \
```

**todo:** We should add [`--strict-channel-priority`](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-channels.html#strict-channel-priority) to this call.

The channel order swap is excellent and follows the [bioconda guidance](https://bioconda.github.io/#do-not-modify-condarc) to have the `conda-forge` channel be higher priority than the `bioconda` channel - we just need to add the other flag.

---

Thanks!  Updated and tested and will merge now.